WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsView.swift

```diff
@@ -0,0 +1,223 @@
+
+    var body: some View {
+        POSPageHeaderView(
+            title: Localization.navigationTitle,
+            trailingContent: {
+                Button(action: { dismiss() }) {
+                    Text(Image(systemName: "xmark"))
+                        .font(.posButtonSymbolLarge)
+                }
+                .foregroundColor(.posOnSurface)
+            })
+        HStack(spacing: POSSpacing.none) {
+            VStack(alignment: .leading, spacing: POSSpacing.none) {
+                List(selection: $selection) {
+                    Section {
+                        ForEach([SidebarNavigation.store, SidebarNavigation.hardware], id: \.self) { item in
+                            HStack {
+                                Image(systemName: item.icon)
+                                    .font(.posBodyLargeRegular())
+                                VStack(alignment: .leading) {
+                                    Text(item.title)
+                                        .font(.posBodyLargeRegular())
+                                    Text(item.subtitle)
+                                        .font(.posBodyMediumRegular())
+                                        .foregroundStyle(.secondary)
+                                }
+                            }
+                            .tag(item)
+                        }
+                    }
+                }
+                .safeAreaInset(edge: .bottom) {
+                    Button {
+                        selection = .help
+                    } label: {
+                        HStack {
+                            Image(systemName: SidebarNavigation.help.icon)
+                                .font(.posBodyLargeRegular())
+                                .foregroundStyle(selection == .help ? .white : .primary)
+                            VStack(alignment: .leading) {
+                                Text(SidebarNavigation.help.title)
+                                    .font(.posBodyLargeRegular())
+                                    .foregroundStyle(selection == .help ? .white : .primary)
+                                Text(SidebarNavigation.help.subtitle)
+                                    .font(.posBodyMediumRegular())
+                                    .foregroundStyle(selection == .help ? .secondary : .secondary)
+                            }
+                            Spacer()
+                        }
+                        .padding(.vertical, POSPadding.small)
+                        .padding(.horizontal, POSPadding.medium)
+                        .contentShape(Rectangle())
+                    }
+                    .buttonStyle(.plain)
+                    .background(
+                        RoundedRectangle(cornerRadius: POSCornerRadiusStyle.large.value, style: .continuous)
+                            .fill(selection == .help ? Color.accentColor : Color.clear)
+                    )
```

Comment:
The selected version looks pretty odd, being purple while the other selected items go grey.

Having rounded corners doesn't work well with the padding we have here either – which is closer to the screen edges because it's outside the list, but if we keep it this way then square corners would make more sense.

I know UI's not final so no need to fix here, but thought I'd mention it.